lib/uname.js

```diff
@@ -0,0 +1,43 @@
+var getUname = require('node-uname')
```

Comment:
Ok, then it's fine for me :-) We could add a `ToDo` comment to check this again in the future if this info gets available in the main builtins :-)